Cargo.toml

```
@@ -23,4 +23,5 @@ crossbeam-channel = "0.5"
 
 [features]
 default = []
```

```suggestion
default = [ "dlopen" ]
```

Ideally I'd suggest having `dlopen` be the default only for Windows and macOS, but I don't think Cargo has a way to specify platform-specific defaults.

---

Considering I wasn't able to get my application to link on Windows without `dlopen`, I definitely think it should be the default.

---

Done.